src/diffpy/structure/tests/__init__.py

```diff
@@ -38,11 +38,7 @@ def testsuite(pattern=""):
     from itertools import chain
     from os.path import dirname
 
-    # Use importlib.resources for Python 3.9 and newer or importlib_resources for older versions
-    try:
-        from importlib import resources as importlib_resources
-    except ImportError:
-        import importlib_resources
+    import importlib_resources
```

Comment:
this doesn't seem to be the righ tbehavior for py > 3.9, no?   I am not sure if you understood my comment?

Reply:
I see the problem. I'm testing it in the wrong way. Let me change them

Reply:
we don't want to use a backport.  We are trying to keep our code at the font edge, not the back edge.  That is my point....